ckanext/ontario_theme/plugin.py

```diff
@@ -46,6 +46,8 @@
 import ckan.lib.uploader as uploader
 import ckan.lib.helpers as h
 
+import ckanext.xloader.interfaces as xloader_interfaces
+
```

Comment:
I don't believe this is used anywhere, is it needed?

Reply:
nopes
left-over from previous experiments